readdirp.js

```diff
@@ -4,8 +4,10 @@ var fs        =  require('graceful-fs')
   , path      =  require('path')
   , minimatch =  require('minimatch')
   , toString  =  Object.prototype.toString
+  , si = typeof setImmediate !== 'undefined' ? setImmediate : function (fn) { setTimeout(fn, 0) }
```

Comment:
Ah, well even though it is just one line, now that it's being used in two places perhaps it would be better to just use https://www.npmjs.com/package/set-immediate-shim instead.